include/zephyr/net/wifi.h

```diff
@@ -470,6 +470,14 @@ static inline const char *wifi_ps_get_config_err_code_str(int16_t err_no)
 	return "<unknown>";
 }
 
+/** Wi-Fi AP mode configuration parameter */
```

Comment:
I just fixed lot of documentation issues in  #72992 and noticed that `@brief` should be added here to get better coverage.
So could you change this to `/** @brief Wi-Fi AP mode configuration parameter */`